rm_vk/rm_vk/Extension/UIViewController+Extension.swift

```diff
@@ -0,0 +1,22 @@
+// UIViewController+Extension.swift
+// Copyright © RoadMap. All rights reserved.
+
+import UIKit
+
+/// Расширение для универсального алерта
+extension UIViewController {
+    // MARK: - Constants
+
+    private enum Constants {
+        static let alertActionTitle = "Ok"
+    }
+
+    // MARK: - Public methods
+
+    func showAlert(title: String, message: String) {
+        let alert = UIAlertController(title: title, message: message, preferredStyle: .alert)
```

Comment:
```suggestion
        let alertController = UIAlertController(title: title, message: message, preferredStyle: .alert)
```
у тебя самой в другой работе так написано